src/gpu/intel/compute/device_info.cpp

```diff
@@ -222,21 +222,21 @@ int device_info_t::max_slm_size(gpu_arch_t gpu_arch) {
     return slm_size;
 }
 
-int device_info_t::max_slm_size_per_tg(gpu_arch_t gpu_arch) {
+int device_info_t::max_slm_size_per_thr(gpu_arch_t gpu_arch) {
```

Comment:
I think this should be left as before as it's SLM size per thread group (not per thread).

Reply:
Done.